spec/requests/tool_integration/contribution_calculator_spec.rb

```diff
@@ -0,0 +1,14 @@
+RSpec.describe ToolMountPoint::ContributionCalculator, type: :request, features: [:workplace_pension_contribution_calculator] do 
```

Comment:
Unless I'm mistaken, the `features` you've specified doesn't match with the feature flag in your routes.